### PR TITLE
Added es_PE person

### PR DESCRIPTION
**Pull request by JsonVladimir**
Added es_PE to person, I recently used the library and noticed that there were no common names from Peru, I decided to propose that the file be added.

### CODE REVIEW

**Review by fcurella — CHANGES_REQUESTED** [review]
Could you add some test cases for the new providers?

**Review by fcurella — CHANGES_REQUESTED** [review]
Can yuou fix the remaining linting issue? You can run `make lint` to format using the project's settings.

**Comment by JsonVladimir** [comment]
> Can yuou fix the remaining linting issue? You can run `make lint` to format using the project's settings.

When I run black it tells me there is 1 file that needs to be reformatted but when i format it it tells me no changes were made.

```
C:\GitHub\faker> python -m black --check --line-length 120 C:\GitHub\faker
would reformat C:\GitHub\faker\tests\providers\test_phone_number.py

Oh no! 💥 💔 💥
1 file would be reformatted, 683 files would be left unchanged.
C:\GitHub\faker> python -m black C:\GitHub\faker\tests\providers\test_phone_number.py
**All done! ✨ 🍰 ✨
1 file _left unchanged_.**
```
image
image

**Comment by fcurella** [comment]
@JsonVladimir I'm not seeing `--line-length 120` on the second command you called. You can just run `make lint` and it should run it with the correct arguments for you

File: tests/providers/test_bank.py [hunk 1]
```diff
@@ -241,6 +242,56 @@ def test_clabe_bank_code(self, faker, num_samples):
...
+    def test_bban(self, faker, num_samples):
+        for _ in range(num_samples):
+            assert re.fullmatch(r"[A-Z]{4}\d{18}", faker.bban())
+
+    def test_iban(self, faker, num_samples):
+        for _ in range(num_samples):
+            iban = faker.iban()
+            assert is_valid_iban(iban)
+            assert iban[:2] == EsPeBankProvider.country_code
+            assert re.fullmatch(r"\d{2}[A-Z]{4}}\d{18}", iban[2:])
```
Comment (fcurella) [inline tests/providers/test_bank.py:257]: I think I see an extra `}` in `{4}}`

**Review by fcurella — CHANGES_REQUESTED** [review]
Could you check the failing tests?

File: tests/providers/test_phone_number.py [hunk 2]
```diff
@@ -285,6 +285,21 @@ def test_phone_number(self, faker, num_samples):
...
+
+    def test_phone_number(self, faker, num_samples):
+        pattern: Pattern = re.compile(
+            r"(\+?51|\(\+51\))?(9\d{8})|"
+            r"(\+?51|\(\+51\))?((9\d{2}) \d{3} \d{3})|"
+            r"(9\d{8})|"
+            r"((9\d{2}) \d{3} \d{3})"
+        )
+        for _ in range(num_samples):
+            phone_number = faker.phone_number_pe()
```
Comment (fcurella) [inline tests/providers/test_phone_number.py:299]: the method is called `phone_number()`:

```suggestion
            phone_number = faker.phone_number()
```

**Review by fcurella — CHANGES_REQUESTED** [review]

File: tests/providers/test_phone_number.py [hunk 3]
```diff
@@ -285,6 +285,12 @@ def test_phone_number(self, faker, num_samples):
             assert pattern.fullmatch(phone_number)
 
 
+class TestEsPE:
+    """Test es_PE phone number provider methods"""
+
+    pass
```
Comment (fcurella) [inline tests/providers/test_phone_number.py:291]: Can we actually test something? :P

**Review by fcurella — CHANGES_REQUESTED** [review]

**Pull request closed without merging**